README.md

```diff
@@ -83,7 +85,42 @@ val consumer = new KafkaConsumer(
 )
 ```
 
+## Jsoniter Scala example
```

Comment:
This file is generated, you should change the file in /doc/src/main/tut and then run
````bash
sbt tut
````

We should pointing out in the README file.